cypress/integration/student-competency-calculations.js

```diff
@@ -37,17 +37,17 @@ describe('Confirm rounding is consistent across UI, API, and exports', () => {
                             cy.wrap(xhr).its('status').should('eq', 200);
                             //convert api baseline to string, if it is not null
                             expect(
-                              apiBaseLine ? `${apiBaseLine}` : apiBaseLine,
```

Comment:
**nitpick**: We shouldn't transform what we pass into `expect`—we want to pass in the output we're checking as closely as possible to how we get it. Instead, transform what we pass to the assertion at the end of the statement

Reply:
Merging for now, but SCHOOL-164 captures following up on this